File: dbutil/connect_db.py
import mysql.connector

mydb = mysql.connector.connect(
  host="localhost",
  port="[PORT]",
  user="[USERNAME]",
  passwd="[PASSWD]",
  database="[DBNAME]"
)

# print(mydb) 

cursor = mydb.cursor()
query = "[SQL STATEMENT]"

cursor.execute(query)
myresult = cursor.fetchall()

for x in myresult:
  print(f"Name: {x[0]} HR: {x[12]}")

mydb.close()

File: dbutil/connect_db.pl
#!/usr/bin/perl
use strict;
use warnings;
use DBI;

my $dbname   = "DBNAME";
my $username = "USERNAME";
my $password = "PASSWORD"

my $dbh = DBI->connect("DBI:mysql:database=$dbname;host=localhost",$username,$password);
die "failed to connect to MySQL database:DBI->errstr()" unless($dbh);

# prepare SQL statement
my $sth = $dbh->prepare("[SQL]") or die "prepare statement failed: $dbh->errstr()";

$sth->execute() or die "execution failed: $dbh->errstr()";

my @row=();

#output database results
while (@row=$sth->fetchrow_array())
{
   print "@row\n";
}

$dbh->disconnect();
print "Done\n";
